docs/HyperIndex/benchmarks.md

```diff
@@ -13,20 +13,20 @@ HyperIndex delivers industry-leading performance for blockchain data indexing. I
 
 ## Recent Independent Benchmarks
 
-The most comprehensive and up-to-date benchmarks were conducted by Sentio in April 2025 and are available in the [sentioxyz/indexer-benchmark repository](https://github.com/sentioxyz/indexer-benchmark). These benchmarks compare Envio's HyperIndex against other popular indexers across multiple real-world scenarios:
+The most comprehensive and up-to-date benchmarks were conducted by Sentio in April 2025 and are available in the [sentio-benchmark repository](https://github.com/enviodev/sentio-benchmark). These benchmarks compare Envio's HyperIndex against other popular indexers across multiple real-world scenarios:
 
 ### Key Performance Highlights
 
 | Case                           | Description                                 | Envio  | Nearest Competitor | TheGraph | Ponder | Advantage vs. Nearest |
 | ------------------------------ | ------------------------------------------- | ------ | ------------------ | -------- | ------ | --------------------- |
 | LBTC Token Transfers           | Event handling, No RPC calls, Write-only    | 2m     | 8m (Sentio)        | 3h9m     | 1h40m  | 4x faster             |
-| LBTC Token with RPC calls      | Event handling, RPC calls, Read-after-write | 1m     | 32m (Subsquid)     | 18h38m   | 4h38m  | 32x faster            |
+| LBTC Token with RPC calls      | Event handling, RPC calls, Read-after-write | 15s    | 32m (Subsquid)     | 18h38m   | 4h38m  | 128x faster           |
 | Ethereum Block Processing      | 100K blocks with Metadata extraction        | 7.9s   | 1m (Subsquid)      | 10m      | 33m    | 7.5x faster           |
 | Ethereum Transaction Gas Usage | Transaction handling, Gas calculations      | 1m 26s | 5m (Subsquid)      | N/A      | 33m    | 3.5x faster           |
 | Uniswap V2 Swap Trace Analysis | Transaction trace handling, Swap decoding   | 41s    | 2m (Subsquid)      | 8m       | N/A    | 3x faster             |
-| Uniswap V2 Factory             | Event handling, Pair and swap analysis      | 20s    | 2m (Subsquid)      | 34m      | 2h24m  | 6x faster             |
+| Uniswap V2 Factory             | Event handling, Pair and swap analysis      | 10s    | 2m (Subsquid)      | 19m      | 2h24m  | 12x faster            |
 
-The independent benchmark results demonstrate that HyperIndex consistently outperforms all competitors across every tested scenario. The most significant performance advantage was seen in real-world indexing scenarios with external RPC calls, where HyperIndex was up to 32x faster than the nearest competitor and over 1000x faster than TheGraph.
+The independent benchmark results demonstrate that HyperIndex consistently outperforms all competitors across every tested scenario. The most significant performance advantage was seen in real-world indexing scenarios with external RPC calls, where HyperIndex was up to 128x faster than the nearest competitor and over 4000x faster than TheGraph.
```

Comment:
Nice!! Seems these are still being very actively developed on so will probably need to revisit and update every now and then